src/sentry/db/models/base.py

```diff
@@ -127,9 +128,19 @@ class Meta:
     __repr__ = sane_repr("id")
 
 
+class DefaultFieldsModel(Model):
+    date_updated = models.DateTimeField(default=timezone.now)
```

Comment:
@MeredithAnya Shouldn't this have `null=True` as well?

Reply:
oh does it have to be? I was thought that we want this to be backfilled as the date the migration was done at.

Reply:
do you just always need to make the column nullable if you are adding it to an already populated table?

Reply:
@MeredithAnya I am not sure about that but I believe so. Also, I think the default for `date_updated` should be `null` since it hasn't actually been saved yet.

Reply:
Since we're setting a default this doesn't have to be nullable. We mostly need to be careful about which tables we add this to though.